test/manifolds/probability_simplex.jl

```diff
@@ -57,6 +57,9 @@ include("../utils.jl")
                 inverse_retraction_methods=[SoftmaxInverseRetraction()],
                 retraction_methods=[SoftmaxRetraction()],
                 test_inplace=true,
+                test_rand_point=true,
+                test_rand_tvector=true,
+                rand_tvector_atol_multiplier=1,
```

Comment:
```suggestion
                rand_tvector_atol_multiplier=10.0,
```
I think I can even trick my remark with a comment and merge it myself, let's try whether this already fixes the tests.